examples/polymer/elements/td-item.html

```diff
@@ -28,49 +35,48 @@
 					},
 					item: {
 						type: Object,
-						value: function() {
-							return {};
-						}
+						value: function() { return {}; }
```

Comment:
They aren't treated differently? There's simply a space after every keyword (`function`, `while`, `if`, ..) It's inconsistent not to have one after function :)